lib/proxy_api/resource.rb

```diff
@@ -63,15 +63,22 @@ def parse(response)
       false
     end
 
+    def escape_path(path)
+      parsed_path = URI.parse(path)
+      query = parsed_path.query
+      path = "#{parsed_path.path}?#{CGI.escape(query)}" unless query.nil?
+      path
+    end
+
     # Perform GET operation on the supplied path
-    def get(path = nil, payload = {})
+    def get(path = nil, additional_headers = {})
```

Comment:
why are we changing the parameter name here?

Reply:
Ewoud ask to change the name

Reply:
If you look at https://rubydoc.info/gems/rest-client/RestClient/Resource#get-instance_method you'll see that it's not really a payload (which would generally be the body) but additional headers. GET never has a body so this makes it match the API it actually calls.